### DISCUSSION TITLE
tinyusb conflicts with flash operation

### DISCUSSION THREAD

**Discussion by arilink** [hathach/tinyusb#1420] [Q&A]
1.Everything works fine when I use FreeRTOS and TinyUSB combination USB works fine
2.I can read and write normally when I use FreeRTOS and Flash to emulate EEPROM
3.When I use the freertos+tinyusb+flash emulated EEPROM in combination, it will trigger HardFault. I try to run the code on Ram or turn off the interrupt temporarily.

**Comment by hathach** [comment]
you need to give more info for reproducing/discussing
1. which board and mcu you are using
2. all software, lib, sdk version
3. link to your simplest code that causes the issue with illustrated on where and how the issue occurs
